assets/i18n/en_US.json

```
@@ -49,7 +49,7 @@
         "widgetTitle": "Patcher",
         "patchButton": "Patch",
         "patchDialogText": "You have selected a resource patch and a split APK installation has been detected, so patching errors may occur.\nAre you sure you want to proceed?",
-        "armv7WarningDialogText": "Patching on ARMv7 devices is not yet supported. Proceed anyways?"
+        "armv7WarningDialogText": "Patching on ARMv7 devices is not yet supported. Patching would very likely fail or abort. If so, you can patch on another device or on your PC. Proceed anyways?"
```

It is not really the warning's job to inform the user about workarounds.
```suggestion
        "armv7WarningDialogText": "Patching on ARMv7 devices is not yet supported and might fail. Proceed anyways?"
```

---

Then they would go to ask for tech support again wasting our time. Isn't the goal of this warning to reduce tech support questions? 

This might not be the best place to put it, but we can't expect every user to join discord and get the suggestion from the bots.

---

This isn't the place to add a workaround. It has to be added somewhere else.

---

I'll remove the workaround but keep the warning about patch failing/aborting

Where should one put the workaround tho?

---

It's not really fitting anywhere in ReVanced Manager.